cdap-docs/reference-manual/source/http-restful-api/metadata.rst

```diff
@@ -372,7 +372,34 @@ user metadata tag, submit an HTTP GET request::
 
 Entities with the specified terms are returned as list of entity IDs::
 
-  ["entity1", "entity2"]
+  [
+      {
+          "entityId": {
+              "id": {
+                  "applicationId": "PurchaseHistory",
+                  "namespace": {
+                      "id": "default"
+                  }
+              },
+              "type": "application"
+          }
+      },
+      {
+          "entityId": {
+              "id": {
+                  "application": {
+                      "applicationId": "PurchaseHistory",
+                      "namespace": {
+                          "id": "default"
+                      }
+                  },
+                  "id": "PurchaseFlow",
+                  "type": "Flow"
+              },
+              "type": "program"
+          }
+      }
```

Comment:
it's ugly...  but accurate... we really need to change the way IDs are represented in the near future.